src/Tizen.NUI/src/public/Layouting/FlexLayout.cs

```diff
@@ -30,9 +31,43 @@ namespace Tizen.NUI
     /// </summary>
     public class FlexLayout : LayoutGroup,  global::System.IDisposable
     {
+        /// <summary>
+        /// FlexAlignmentSelfProperty
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static readonly BindableProperty FlexAlignmentSelfProperty = BindableProperty.CreateAttached("FlexAlignmentSelf", typeof(AlignmentType), typeof(FlexLayout), AlignmentType.Auto, propertyChanged: OnChildPropertyChanged);
```

Comment:
good job! (b^_^)b

https://docs.microsoft.com/ko-kr/dotnet/api/xamarin.forms.bindableproperty.createattached?view=xamarin-forms

one question...
after NUI adapt xamarine xaml, some properties were changed as bindable property like this. this kind of bindable property is created at the moment of being called and used? I am concerned about the launching speed and performance degration... ^^;

Reply:
> good job! (b^_^)b
> 
> https://docs.microsoft.com/ko-kr/dotnet/api/xamarin.forms.bindableproperty.createattached?view=xamarin-forms
> 
> one question...
> after NUI adapt xamarine xaml, some properties were changed as bindable property like this. this kind of bindable property is created at the moment of being called and used?

Since `FlexLayout` don't have static constructor, bindable proeprties are created at the moment of being called and used.

> I am concerned about the launching speed and performance degration... ^^;

I guess we can use `Dictionary` for support attached property instead of `BindableProperty`. but, i'm not sure that it is possible to use in xaml.